tg/MainScreenListOfGames/ListOfGamesViewController.swift

```diff
@@ -104,7 +104,7 @@ class ListOfGamesViewController: UIViewController {
                 }
             }
             
-            guard let id = item.id else { continue }
+            let id = item.id
```

Comment:
Unneeded string

Reply:
String is still here

Reply:
dictionaryOfLogo[item.id] = readyImage, it's better. Don't create additional constants if there is no reason for it